tjreports/site/views/reports/tmpl/default.php

```diff
@@ -221,7 +221,7 @@
 						}
 						?>
 					<!--/col-md-2-->
-							<div class="show-hide-cols col-md-2 col-sm-3">
+							<div class="show-hide-cols col-md-2 col-sm-3" style="width: min-content;">
```

Comment:
can we avoid inline css, and use classes? @MeghaBiranje

Reply:
Change some code